web/src/lib/types/user.ts

```diff
@@ -173,6 +173,7 @@ export function badgeFromJson(json: any): Badge {
 
 export class User implements IAccount {
     identity: string = '';
+    nostrPublicKey: string | null = null;
```

Comment:
For now I think this is ok, but we'll need to think about this soon. We could end up with a (PM generated) private key, a public key that the user introduced (but cannot be used for most Nostr functionalities because we don't have the related private key), and the user using a third identity (or fourth, or...) using browser extensions.